example/src/View.elm

```diff
@@ -70,3 +72,26 @@ formView form =
                     [ text "Reset" ]
                 ]
             ]
+
+
+linksView : Form CustomError User -> Html Form.Msg
+linksView form =
+    let
+        linkView i =
+            div
+                [ class "row link" ]
+                [ col' 3 [ text "Link" ]
+                , Input.textInput
+                    (Form.getFieldAsString ("links." ++ (toString i) ++ ".name") form)
```

Comment:
The link input field's CSS seems to be missing.

![elm_simpleform_example](https://cloud.githubusercontent.com/assets/125707/19839528/433bc278-9eec-11e6-80b6-789d7f6db153.jpg)